### PR TITLE
Improving the ISSUE_TEMPLATES

### PR DESCRIPTION
**Pull request by MrRowey**
Added Bug and Feature Form Template
I updated the Engine Crash template to Form-based.

### CODE REVIEW

**Comment by sprunk** [comment]
what's wrong?

**Comment by MrRowey** [comment]
It was meant to mark as a draft. The mobile app must have played up.

**Comment by MrRowey** [comment]
I need to Update the Text as theses are Base Template I Created for FAF

File: .github/ISSUE_TEMPLATE/2-feature_request.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time in making a Feature Request.
+
+        - This repository is maintained by volunteers, it may take a while for them to come back to you.
+        - Remember to include sufficient details and context.
+        - If you have multiple Idea's or suggestions, please submit them in separate issues.
+        - We accept pull request. If you know how to fix it - go ahead!
```
Comment (sprunk) [inline .github/ISSUE_TEMPLATE/2-feature_request.yml:15]: Would be good to make it clear that it's good to make sure a PR is wanted in the first place (by talking to a maintainer or making a ticket). Not every feature is desirable at engine level

File: .github/ISSUE_TEMPLATE/2-feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    attributes:
+      label: Describe the Feature
+      description: A clear and concise description of what the feature is.
+    validations:
+      required: true
+  - type: textarea
+    id: behavior
+    attributes:
+      label: Expected Behavior
+      description: If applicable, a clear and concise description of what you expect to happen in various scenario's.
```
Comment (sprunk) [inline .github/ISSUE_TEMPLATE/2-feature_request.yml:27]:
```suggestion
      description: If applicable, a clear and concise description of what you expect to happen in various scenarios.
```

File: .github/ISSUE_TEMPLATE/3-engine_crash.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting a bug! This template will help you with creating a report that we can use to understand the bug.
+          - This repository is maintained by volunteers, it may take a while for them to come back to you.
+          - Remember to include sufficient details and context.
+          - If you have multiple questions, suggestions, or bugs, please submit them in separate issues.
+          - We accept pull request. If you know how to fix it - go ahead!
```
Comment (sprunk) [inline .github/ISSUE_TEMPLATE/3-engine_crash.yml:14]:
```suggestion
          - We accept pull requests. If you know how to fix it - go ahead!
```

File: .github/ISSUE_TEMPLATE/3-engine_crash.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  - type: textarea
+    attributes:
+      label: URL of infolog.txt
```
Comment (sprunk) [inline .github/ISSUE_TEMPLATE/3-engine_crash.yml:17]: what about infologs with no URL (i.e. a loose file somebody has on their PC)?

File: .github/ISSUE_TEMPLATE/3-engine_crash.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      description: Please Provide the crash log located for the engine.
+      placeholder: https://server3.beyondallreason.info/teiserver/reports/infolog/8117
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Engine Version
+      description: Please Provide the Version of the engine the crash happened on.
```
Comment (sprunk) [inline .github/ISSUE_TEMPLATE/3-engine_crash.yml:25]:
```suggestion
      description: Please provide the version of the engine the crash happened on.
```
Would be good to explain (or link to a guide) on how to figure out the version.

File: .github/ISSUE_TEMPLATE/3-engine_crash.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+      placeholder: 105.1.1-1354-g72b2d55 bar
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Stack Trace
+      description: Translated Stack Trace with links
+      placeholder: Translated goes here, with links!
```
Comment (sprunk) [inline .github/ISSUE_TEMPLATE/3-engine_crash.yml:33]: translated how?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Beyond All Reason Discord Server
+    url: https://discord.gg/beyond-all-reason
```
Comment (sprunk) [inline .github/ISSUE_TEMPLATE/config.yml:4]: could also have a link to the general recoil server

**Pull request closed without merging**